packages/ui/components/MenuActionButton.tsx

```
@@ -0,0 +1,21 @@
+  className = "",
+}: {
+  icon: React.ElementType;
+  label: string;
+  onClick?: () => void;
+  className?: string;
+}) {
+  return (
+    <button
+      className={`cursor-pointer flex items-center text-color px-2.25 py-1 w-full text-left duration-200 opacity-50 hover:opacity-65 ${className}`}
+      onClick={onClick}
+    >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)